clients/gtest/CMakeLists.txt

```diff
@@ -99,3 +94,29 @@ endif( )
 
 set_target_properties( hipsolver-test PROPERTIES DEBUG_POSTFIX "-d" CXX_EXTENSIONS NO )
 set_target_properties( hipsolver-test PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging" )
+
+if(WIN32)
+  file(GLOB third_party_dlls
+    LIST_DIRECTORIES OFF
+    CONFIGURE_DEPENDS
+    ${cblas_DIR}/bin/*.dll
+    ${GTest_DIR}/bin/*.dll
+    $ENV{rocblas_DIR}/bin/*.dll
+	$ENV{rocsolver_DIR}/bin/*.dll
```

Comment:
Minor issue: This indent is a bit odd. Also, github is complaining there's no newline at the end of the file.

Reply:
I'm missing my usual tools when I work on Windows... Thanks for catching that. Fixed.